API.md

```diff
@@ -41,7 +41,7 @@ getAccessToken(config)
 #### Arguments
 
 * [`config: OAuthConfig`](#tbd) - OAuth configuration for the request (specify grant type and corresponding parameters)
-* [`logger?: Logger`](#tbd) - Optional logger
+* [`logger?: Logger`](#tbd) - optional logger
```

Comment:
actually we decided to leave 'optional' out because we have the `?`syntax

Reply:
👍